src/processes.c

```diff
@@ -843,6 +856,11 @@ static void ps_submit_proc_list(procstat_t *ps) {
   vl.values_len = 1;
   plugin_dispatch_values(&vl);
 
+  sstrncpy(vl.type, "ps_pss", sizeof(vl.type));
+  vl.values[0].gauge = ps->vmem_pss;
```

Comment:
Below, `vmem_pss` is set to a well-known value on error. Add a check for this well-known value here and set the gauge to `NAN` if there was an error:

```c
vl.values[0].gauge = ps->vmem_pss;
if (ps->vmem_pss == 0) {
  vl.values[0].gauge = NAN;
}
```